README.md

```diff
@@ -1 +1,60 @@
-# school-library
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# School Library
+
+> This is basic school library app that has been built with Ruby. The app allows for some basic operations  for example getting the name of the student or the teacher as well as constraints that come with their age.
+
+The app will allow you to:
+
+- Add new students or teachers.
+- Add new books.
+- Save records of who borrowed a given book and when.
+- And all of this will be built in a beautiful and well-organized way!
+
+## Built With
+
+- Ruby
+
+
+
+
+## Getting Started
+
+Clone this repository using
+`git clone https://github.com/Nuri1977/school-library.git`
+
+cd into the repository
+`cd school-library`
+
+run files with
+`ruby FILENAME`
+
+
+
+## Authors
+
+👤 **Nuri LAcka**
+
+- GitHub: [@Nuri1977](https://github.com/Nuri1977)
+- Twitter: [@Lackanuri](https://twitter.com/LackaNuri)
+- LinkedIn: [@nurilacka](https://www.linkedin.com/in/nuri-lacka-7141b01ba/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] [OPTIONAL] The link to the MIT license is not working please have another look at [ the template](https://github.com/microverseinc/readme-template) for the readme file provided by Microverse and make sure the link to the MIT license works the same as in that template.